test/new_relic/agent/log_event_aggregator_test.rb

```diff
@@ -324,6 +328,35 @@ def test_records_metrics_on_harvest
       end
     end
 
+    def test_seen_and_sent_zeroed_out_on_reset!
```

Comment:
Since I updated `reset!`, I thought it could use more testing for its other functionality.